patches/minecraft/net/minecraft/entity/AgeableEntity.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/entity/AgeableEntity.java
++++ b/net/minecraft/entity/AgeableEntity.java
+@@ -135,6 +135,7 @@
+    }
+ 
+    protected void func_175500_n() {
++      net.minecraftforge.event.ForgeEventFactory.onAgeableEntityGrowingAdult(this);
```

Comment:
Hooglins don't call super so this wont fire for them.

Reply:
Added event to `HoglinEntity.onGrowingAdult()` also